src/components/LanguageToggle.js

```diff
@@ -14,8 +14,9 @@ const Dropdown = styled.div`
 `;
 
 function LanguageToggle() {
-  const { lang, dispatch } = useContext(AppContext);
-  const { languages } = useContext(LanguagesContext);
+  const { lang, languages, dispatch } = useContext(AppContext);
```

Comment:
you don't need to read languages from the context

Reply:
Just use the imported file directly

Reply:
`import languges from '../i18n/locales'
`

Reply:
OH, My bad, i sticked with the context idea haha